### ISSUE TITLE
vue create puglin installation broken

### ISSUE THREAD

**Issue by starskyC** [vuejs/vue-cli#5547]
### Version
@vue/cli 4.4.1

### Environment info
```bash
  System:
    OS: Windows 10 10.0.18363
    CPU: (12) x64 AMD Ryzen 5 2600 Six-Core Processor
  Binaries:
    Node: 12.16.1 - C:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 6.14.5 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 44.18362.387.0
  npmPackages:
    @vue/cli-overlay:  4.4.1
    @vue/cli-plugin-eslint: ^4.4.1 => 4.4.1
    @vue/cli-plugin-pwa: ~4.4.0 => 4.4.1
    @vue/cli-plugin-router: ~4.4.0 => 4.4.1
    @vue/cli-plugin-unit-mocha: ~4.4.0 => 4.4.1
    @vue/cli-plugin-vuex: ~4.4.0 => 4.4.1
    @vue/cli-service: ~4.4.0 => 4.4.1
    @vue/cli-shared-utils:  4.4.1
    @vue/component-compiler-utils:  3.1.2
    @vue/preload-webpack-plugin:  1.1.1
    @vue/test-utils: ^1.0.3 => 1.0.3
    @vue/web-component-wrapper:  1.2.0
    eslint-plugin-vue: ^6.2.2 => 6.2.2
    vue: ^2.6.11 => 2.6.11
    vue-eslint-parser:  7.1.0
    vue-hot-reload-api:  2.3.4
    vue-loader:  15.9.2
    vue-router: ^3.2.0 => 3.3.2
    vue-style-loader:  4.1.2
    vue-template-compiler: ^2.6.11 => 2.6.11
    vue-template-es2015-compiler:  1.9.1
    vuex: ^3.4.0 => 3.4.0
```

### Steps to reproduce
1. install vue cli globally using npm
2.` vue create my-vue-app`
3. manually select some random plugins
4. `cd my-vue-app && npm run serve`

### What is expected?
The app should work fine since it's created from CLI directly and I haven't done anything yet

### What is actually happening?
All the plugins are missing, and couldn't be solved using either
`npm i <missing-plugin-name>`
or
`vue add <missing-plugin-name>`

---
Later I tested creating with no plugins, thinking that I can first run the app without any plugin and see if there is any way adding them afterward. Then these came out:
```bash
 error  in ./src/App.vue

Module Error (from C:/Users/Administrator/AppData/Roaming/npm/node_modules/@vue/cli-service/node_modules/vue-loader/lib/index.js):
[vue-loader] vue-template-compiler must be installed as a peer dependency, or a compatible compiler implementation must be passed via options.
```

**Comment by sodatea** [comment]
You must have pre-configured `NODE_ENV=production` in your environment. Please check that and unset it.

**Comment by starskyC** [comment]
I found the problem! The default preset assumes `vue-cli-service` to be installed globally, and it wasn't. Then after I installed it globally(it's useless to installed `-d`, because there is one already installed but couldn't be detected), the global `vue-cli-service` seems like couldn't find any of the plugin. I think this explains. So now I change the scripts into
`"serve": "node_modules/.bin/vue-cli-service serve"`

it was this before I change.
`"serve": "vue-cli-service serve"`

same thing with `build` script
